.infrastructure/namespace.yml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+ name: todoapp
+ 
```

Comment:
![image](https://github.com/user-attachments/assets/fb0aa9d8-c1b1-4218-85d0-e3d81ef33030)
![image](https://github.com/user-attachments/assets/3cfb2b77-c07a-4e66-8bec-162eeac5e0fc)

For EOF to display correctly, the line must be empty without spaces.